benchmarks/mwt/workload-large/00-mwt-workload/00-install-workload.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl apply -f ./workload.yaml
```

Comment:
this is another strange apply... we can use the `apply:` property... or place the file in this file.

Reply:
same, workload was deleted right after